graphql/queries.graphql

```diff
@@ -71,6 +71,28 @@ query GridsQuery {
   }
 }
 
+query FinishedGridsQuery {
+  grid(order_by: {test_initiation_date: desc}, where: {finished: {_eq: true}}) {
+    id
+    title
+    sample_arrival_date
+    sample_taken_date
+    test_finished_date
+    test_initiation_date
+    finished
+  }
+}
+
+query FinishedLabResultQuery($gridIds: [uuid!]!) {
+  lab_result(order_by: {updated_at: asc}, where: {referenced_in_grid_id: {_in: $gridIds}}) {
```

Comment:
Shouldn't it be descending? So we pick the last one?

It took me some time to understand this but we need `asc`, so that in `keyBy` function we override the `sample_code` with the latest entry, right? 

This sound like a bit of spaghetti to me, but I guess for now it's all right, just add TODO about this (maybe before the `keyBy` fn). Or did I misunderstood?

Reply:
honestly, I want to throw out the whole getEntries fn - instead, we can just pull appropriate data for each table separately and be much clearer in what the app shows

yup, this is horrible but it was done in about 20 minutes on a best-effort basis :)

Reply:
Yeah, getting rid of that is surely a good idea. Right now, we are changing the logic of the shown entries every two days, so we can refactor when it's clearer what we want to show :)